frontend/src/styles/common.scss

```diff
@@ -431,7 +431,7 @@ html {
     cursor: pointer;
 }
 
-.dialog-footer{
+.dialog-footer {
     display: flex;
     align-items: center;
     justify-content: flex-end;
```

Comment:
I'm sorry for confusion but I cannot analyze or compare two different versions of a file since you haven't attached them to my conversation with you. Please attach the files so that I can assist you.